packages/react-app/src/types/Role.ts

```diff
@@ -1,2 +1,34 @@
-const roles = ['create-bounty', 'edit-bounty', 'all'] as const;
-export type Role = typeof roles[number];
\ No newline at end of file
+// list of roles in the application
+export type Role =
```

Comment:
💯 for activity based auth patterns. 

when we start working on claim, will this start to map more closely to the activities (create, claim, submit, complete) ?

Reply:
Yeah, can go as granular as we need !